packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -1446,4 +1446,18 @@ exports.examples = [
       );
     },
   },
+  {
+    title: 'TextInput with aria-labelledby attribute"',
+    render(): React.Element<typeof View> {
+      return (
+        <View>
+          <Text nativeID="testAriaLabelledBy">Phone Number</Text>
+          <TextInput
+            aria-labelledby={'testAriaLabelledBy'}
```

Comment:
Where is the support for this implemented in TextInput, Text, Image components?

Reply:
@necolas I was checking this to update the docs and it seems that `Text` doesn't ever support `accessibilityLabelledBy`

Reply:
Oh good to know

Reply:
I've just opened a PR removing it, https://github.com/facebook/react-native/pull/35327